### ISSUE TITLE
Request: Adding Half Precision/fp16 statement

### ISSUE THREAD

**Issue by mastrcheif** [facebook/fb.resnet.torch#81]
It would be nice to boost performance on the jetson tx1 with Floating Point 16 for classifying an image with that feature.

**Comment by colesbury** [comment]
Yes, this would be nice. We need to wait for some PRs to be merged into cutorch and cudnn.torch first.

**Comment by mastrcheif** [comment]
It seems like they have merged the required PR for it the other day =) 
[cudnn.cutorch#79](https://github.com/soumith/cudnn.torch/issues/79)
